<!DOCTYPE html>

<html lang="en" class="gr__cseweb_ucsd_edu"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <title> STELAR </title>

  <!-- Bootstrap core CSS -->
  <link href="website/css" rel="stylesheet" type="text/css">
  <link href="website/bootstrap.min.css" rel="stylesheet">
  <link href="website/jumbotron-narrow.css" rel="stylesheet">
  <link href="website/main.css" rel="stylesheet">



  <script src="website/jquery-1.11.0.min.js"></script>
  <script src="website/bootstrap.min.js"></script>
  <script>
    $(document).ready(function() {
      // add a hash to the URL when the user clicks on a tab
      $('a[data-toggle="tab"]').on('click', function(e) {
	history.pushState(null, null, $(this).attr('href'));
      });
      // navigate to a tab when the history changes
      window.addEventListener("popstate", function(e) {
	var activeTab = $('[href=' + location.hash + ']');
	if (activeTab.length) {
	  activeTab.tab('show');
	} else {
	  $('a[href=#about]').tab('show');
	}
      });
      $('a[href=#about]').tab('show');
    })
  </script>
  
  
 
  </head>

 


<div class="container">
  <div class="header">
    
  <ul class="nav nav-pills pull-right" id="tabs">
      <li class="active"><a href="#about" data-toggle="tab">about</a></li>
      <li class=""><a href="#installation" data-toggle="tab">installation</a></li>
      <li class=""><a href="#datasets" data-toggle="tab">datasets</a></li>
     <!-- <li class=""><a href="#contributors" data-toggle="tab">contributors</a></li> !-->
  </ul>
  <h3><strong>STELAR</strong></h3>
<div class="tab-content">


<div class="tab-pane active" id="about">
<div class="col-lg-12">
  
<h4>Description</h4>
<p>
STELAR is a Java program based on phylonet <a href="https://bioinfocs.rice.edu/phylonet"> (bioinfo.cs.rice.edu/phylonet) </a> for estimating species trees given a set of rooted gene trees, such that duplications or duplication and losses are minimized (within a constrained search space). 
The algorithm used is described in our paper title <span class="paper-title">"STELAR: A statistically consistent coalescent-based species trees estimation method by maximizing triplet consistency"</span>.
</p>

<h4>Acknowledgment</h4>
<p>
STELAR code uses code from the PhyloNet package (by Luay Nakhleh).
</p>
<h4>Bug Reports</h4>
<p>
We are always looking for ways to improve our codes. For any bugs please email at:  mazhar.buet11@gmail.com.
</p>
  <h4>Background</h4>
  <p>
  Species tree estimation is frequently based on phylogenomic approaches that use multiple genes from throughout
  the genome. However, estimating a species tree from a collection of gene trees can be complicated due to the presence of gene tree incongruence resulting from incomplete lineage sorting (ILS), 
  which is modelled by the multi-species coalescent process. Maximum likelihood and Bayesian MCMC methods can potentially result in accurate trees, but they do not scale well to large datasets.
  </p>
  
  <h4> Results </h4>
  <p> 
  We present STELAR (Species Tree Estimation by maximizing tripLet AgReement), a new statistically consistent coalescent-based method for estimating species trees from a collection of gene trees. 
  We formalized the constrained triplet consensus (CTC) problem and showed that the solution to the CTC problem is a statistically consistent estimate of the species tree under the multi-spcies coalesccent model. 
  STELAR is an efficient dynamic programming based solution to the CTC problem which is very fast and highly accurate. We evaluated the accuracy of STELAR in comparison with MP-EST and ASTRAL -- two of the most popular and accurate coalescent based methods. 
  Experimental results suggest that STELAR matches the accuracy of ASTRAL and is more accurate than MP-EST.
  </p>
  <h4> Conclusions </h4>
  <p>
  Theoretical and empirical results (on both simulated and real biological datasets) suggest that  STELAR is a valuable technique for species tree estimation from gene tree distributions.
  </p>
  
  <h4>Availability </h4> 
  <p>
  STELAR is freely available at <a href="https://github.com/islamazhar/STELAR">https://github.com/islamazhar/STELAR</a> 
  </p>

</div>
</div>

<div class="tab-pane" id="installation">
<div class="col-lg-12">
  <h3>Installtion guide</h3>
</div>
</div>


<div class="tab-pane" id="datasets">
  <div class="col-lg-12">
    <h3>Datasets</h3>
    We have analyzied the following datasets.
    <h4>11 taxon</h4>
    <h4>15 taxon</h4>
    <h4>37 taxon</h4>
    <h4>Real biological dataset</h4>
  </div>
</div>


</div> <!-- tab-content -->
</div> <!-- container -->




</body><span class="gr__tooltip"><span class="gr__tooltip-content"></span><i class="gr__tooltip-logo"></i><span class="gr__triangle"></span></span></html>
